Class/practice2.1/task3.cpp

```diff
@@ -0,0 +1,84 @@
+#include <iostream>
+#include <fstream>
+#include <unordered_map>
+#include <string>
+
+class UserStatsManager {
+private:
+    const std::string DATABASE_FILE = "user_stats.txt";
+    const std::string SECRET_WORD = "bread";
+    std::unordered_map<std::string, int> userStats;
+
+    void loadDatabase() {
+        std::ifstream infile(DATABASE_FILE);
+        if (!infile.is_open()) return;
+
+        std::string name;
+        int count;
+        while (infile >> name >> count) {
+            userStats[name] = count;
+        }
+
+        infile.close();
+    }
+
+    void saveDatabase() const {
+        std::ofstream outfile(DATABASE_FILE);
+        for (const auto& entry : userStats) {
```

Comment:
we can uuse more readable alternative for (const auto& [name, count] : userStats)